packages/admin/src/middleware.page.ts

```diff
@@ -11,7 +11,17 @@ const authenticateRequest = async (req: NextRequest, res: NextResponse) => {
   const authCookie = req.cookies.get('session_id');
   const userJwtCookie = req.cookies.get(process.env.JWT_COOKIE_NAME);
 
-  if (!authCookie || !userJwtCookie) {
+  await csrfMiddleware(req, res);
```

Comment:
hmm git has been doing some strange merges today - this line needs to be removed or we'll end up calling the middleware twice, which does not work as I discovered yesterday